src/garage/envs/pybullet/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Wrappers for the Pybullet based gym environments.
+
+See https://github.com/bulletphysics/bullet3/tree/master/examples/pybullet
+"""
+try:
+    import pybullet_envs  # noqa: F401
+except Exception as e:
+    raise ImportError("To use garage's pybullet wrappers, please install "
+                      'garage[pybullet].')
```

Comment:
Please stick to single quotes

also, i think you should remove the trailing period, since some people might get confused copy-pasting.